fbw-a380x/src/systems/instruments/src/EWD/EWD.tsx

```diff
@@ -77,6 +85,11 @@ export class EngineWarningDisplay extends DisplayComponent<{ bus: ArincEventBus
     this.engineStateSubs[2].setConsumer(sub.on('engine_state_3').whenChanged());
     this.engineStateSubs[3].setConsumer(sub.on('engine_state_4').whenChanged());
 
+    this.engFirePb[0].setConsumer(sub.on('engine_fire_pb_1').whenChanged());
+    this.engFirePb[1].setConsumer(sub.on('engine_fire_pb_2').whenChanged());
+    this.engFirePb[2].setConsumer(sub.on('engine_fire_pb_3').whenChanged());
+    this.engFirePb[3].setConsumer(sub.on('engine_fire_pb_4').whenChanged());
```

Comment:
Redundant. I also wonder what's the advantage of putting these in an array since no work is done on the array. It would be clearer to have `engine1FirePb` rather than `engFirePb[0]` IMO.
```suggestion
    this.engFirePb[0].setConsumer(sub.on('engine_fire_pb_1'));
    this.engFirePb[1].setConsumer(sub.on('engine_fire_pb_2'));
    this.engFirePb[2].setConsumer(sub.on('engine_fire_pb_3'));
    this.engFirePb[3].setConsumer(sub.on('engine_fire_pb_4'));
```